client/src/modules/stock/aggregated_consumption/templates/lot_movement_date.tmpl.html

```diff
@@ -0,0 +1,7 @@
+<div class="ui-grid-cell-contents">
+  <bh-date-picker
```

Comment:
I think this datepicker should have `min` and `max` dates within the period, right?